docs/en_US/Release_v1.0.md

```diff
@@ -0,0 +1,33 @@
+
+<p align="center">
+<img src=".././img/release-1-title-1.png" width="100%" />
+</p>
+
+
+From September, 2018 to September, 2019, We are still moving on …
+
+
+ **Great news!**&nbsp;&nbsp;With the tag of **Scalability** and **Ease of Use** , NNI v1.0 is comming. Based on the various types of [Tuning Algorithms](./Tuner/BuiltinTuner.md), NNI has supported the [Neural Architecture Search](docs/en_US/AdvancedFeature/GeneralNasInterfaces.md) and  Auto-Feature generator & selection,which is an exciting news for algorithmic engineers, besides these, NNI v1.0 has made many improvements in the optimization of tuning algorithm,  [WebUI's simplicity and intuition](./Tutorial/WebUI.md) and [platform diversification](./TrainingService/SupportTrainingService.md). NNI has grown into a more intelligent automated machine learning(AutoML) toolkit.
```

Comment:
Should be ";" before besides. Space before `(AutoML)`. Missing space after `Auto-Feature generator & selection,`. Why are some words separated with two spaces?

Reply:
ok ,all fixed. TT

Reply:
I suggest change the “Auto-Feature generator & selection" to "Auto-Feature-Engineering"

Reply:
ok, fixed~